zulipterminal/ui_tools/boxes.py

```diff
@@ -71,10 +71,14 @@ def __init__(self, view: Any) -> None:
 
         # Used in PM and stream boxes
         # (empty list implies PM box empty, or not initialized)
-        # * prioritizes autocomplete in message body
-        # * updates server on PM typing events
+        # Prioritizes autocomplete in message body
         self.recipient_user_ids: List[int] = []
 
+        # Updates server on PM typing events
+        # Is separate from recipient_user_ids because we
+        # don't include the user's own id in this list
+        self.typing_recipients: List[int] = []
```

Comment:
This is nice simple extra variable which works fine. My only thoughts are:
* Maybe append `_user_ids` to indicate more clearly what's in it elsewhere
* Should it be reset everywhere that recipient_user_ids is? It doesn't currently appear to be? (I know they have different uses, but it seems useful to be consistent)
* If we're resetting/setting at similar times, should that be in a function, existing or otherwise? (maybe a refactor?)